apis/fluentd/v1alpha1/plugins/common/common_types.go

```diff
@@ -20,10 +20,10 @@ type CommonFields struct {
 
 // Time defines the common parameters for the time plugin
 type Time struct {
-	// parses/formats value according to this type, default is *string
-	// +kubebuilder:validation:Enum:=float;unixtime;*string;mixed
+	// parses/formats value according to this type, default is string
+	// +kubebuilder:validation:Enum:=float;unixtime;string;mixed
 	TimeType *string `json:"timeType,omitempty"`
-	// Process value according to the specified format. This is available only when time_type is *string
+	// Process value according to the specified format. This is available only when time_type is string
```

Comment:
should change like this?
```suggestion
	// Process value according to the specified format. This is available only when time_format is string
```